tests/test_upload.py

```diff
@@ -101,177 +101,11 @@ def test_get_key_content_type(settings):
     assert get_key_content_type("foo.HTML") == "text/html"
 
 
-@pytest.mark.django_db
-def test_upload_archive_happy_path(
```

Comment:
The S3 variant of ``test_upload_archive_happy_path`` appears later in the file. To try to keep the diff sane for review, I removed this GCS variant test and left the S3 variant where it was.